### ISSUE TITLE
Making table header sticky when scrolling the page

### ISSUE THREAD

**Issue by edirpedro** [hestiacp/hestiacp#1487]
Just a CSS trick to make the table header fix on top when scrolling the page, useful when reading too much data on screen.  Maybe this could be added.

```
.table-header {
    position: sticky;
    top: 145px; /* height size of the fixed header */
}
```

![Captura de Tela 2020-12-18 às 11 40 02](https://user-images.githubusercontent.com/831507/102626720-f1210300-4125-11eb-831b-4d7b56c15e0b.png)

Another option, better and cleaner I think, should be duplicate the .table-header element at the bottom of the list and then add the sticky atribute to it.

```
.table-footer {
    position: sticky;
    bottom: 0;
}
```

![Captura de Tela 2020-12-18 às 14 14 56](https://user-images.githubusercontent.com/831507/102641920-a8744480-413b-11eb-9fc6-fd8e7ddfa4aa.png)

**Comment by jaapmarcus** [comment]
Feel free to create an pull request 

https://github.com/hestiacp/hestiacp/blob/main/web/css/src/styles.css

and the minified version
https://github.com/hestiacp/hestiacp/blob/main/web/css/styles.min.css

**Comment by ghost** [comment]
Hi @edirpedro, thanks for submitting this to us. I'll implement this for the next major release. :)

**Comment by ghost** [comment]
Hi @edirpedro, this change has been implemented for v1.4.0 which will be available soon.
